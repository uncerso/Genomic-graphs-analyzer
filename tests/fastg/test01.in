>EDGE_3_length_1_cov_1.000000;
>EDGE_2_length_1_cov_1.000000;
>EDGE_1_length_1_cov_1.000000:EDGE_2_length_1_cov_1.000000,EDGE_3_length_1_cov_1.000000;
