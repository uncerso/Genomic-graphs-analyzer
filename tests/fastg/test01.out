Info about weakly connected components:

Amount of vertices in a component : amount of components
4 : 1

Amount of edges in a component : amount of components
3 : 1

Total length of edges in a component : amount of components
3 : 1
